source/includes/wp-api-v2/_customers.md

```diff
@@ -4,29 +4,24 @@ The customer API allows you to create, view, update, and delete individual, or a
+| `date_created`       | date-time | The date the customer was created, in the site's timezone. <i class="label label-info">read-only</i>                                                                                                                                                                                                                                                               |
+| `date_modified`      | date-time | The date the customer was last modified, in the site's timezone. <i class="label label-info">read-only</i>                                                                                                                                                                                                                                                         |
+| `email`              | string    | The email address for the customer. <i class="label label-info">mandatory</i>                                                                                                                                                                                                                                                                                      |
+| `first_name`         | string    | Customer first name.                                                                                                                                                                                                                                                                                                                                               |
+| `last_name`          | string    | Customer last name.                                                                                                                                                                                                                                                                                                                                                |
+| `role`               | string    | Customer role. <i class="label label-info">read-only</i>                                                                                                                                                                                                                                                                                                           |
+| `username`           | string    | Customer login name. Can be generated automatically from the customer's email address if the option `woocommerce_registration_generate_username` is equal to `yes` <i class="label label-info">cannot be changed</i> <i class="label label-info">maybe mandatory</i>                                                                                               |
```

Comment:
It's maybe, but we say "Can be generated automatically from the customer's email address if the option `woocommerce_registration_generate_username` is equal to `yes`". At the same time, if you check the `OPTIONS` for it, will return if is mandatory or not on your site.